tests/conftest.py

```diff
@@ -1673,15 +1674,23 @@ async def _async_call(
         domain: str,
         service: str,
         service_data: dict[str, Any] | None = None,
-        **kwargs: Any,
+        blocking: bool = False,
+        context: Context | None = None,
+        target: dict[str, Any] | None = None,
+        return_response: bool = False,
     ) -> ServiceResponse:
-        calls.append(ServiceCall(domain, service, service_data))
+        calls.append(
+            ServiceCall(domain, service, service_data, context, return_response)
+        )
```

Comment:
`geo_location` tests the `Context` of the `ServiceCall`, so making sure it is set here in the fixture.